sql/core/src/test/scala/org/apache/spark/sql/ShowCreateTableSuite.scala

```diff
@@ -200,6 +200,32 @@ abstract class ShowCreateTableSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("SPARK-37494: Unify v1 and v2 option output") {
+    withTable("ddl_test") {
+      sql(
+        s"""CREATE TABLE ddl_test (
+           |  a STRING
+           |)
+           |USING json
+           |TBLPROPERTIES (
+           | 'a' = '1',
+           | 'b' = '2')
+           |OPTIONS (
+           | 'k1' = 'v1',
+           | k2 = 'v2',
+           | `k3` 'v3',
+           | k4 'v4',
```

Comment:
can we randomly order the properties and options to test the ordering fix?

Reply:
sorry.